skimage/measure/_regionprops.py

```diff
@@ -693,7 +839,13 @@ def regionprops(label_image, intensity_image=None, cache=True,
             0.15 and earlier. However, for some properties, the transformation
             will be less trivial. For example, the new orientation is
             :math:`\frac{\pi}{2}` plus the old orientation.
-
+    extra_properties : Iterable of callables
+        Add extra property computation functions that are not included with
+        skimage. The name of the property is derived from the function name,
+        the dtype is inferred by calling the function on a small sample.
+        If the name of an extra property clashes with the name of an existing
+        property the extra property wil not be visible and a UserWarning is
+        issued.
```

Comment:
Shouldn't we specify that the first argument of the property function must be the mask of the considered region and the optional second argument must be the corresponding image?

Reply:
yes, we should. So far that could only be derived from the examples. I just added this, happy for suggestions about how to phrase it better.